### PR TITLE
globalgroups: Update with MediaWiki latest core/extensions permissions

### PR DESCRIPTION
**Pull request by MarcoAurelioWM**
* Updated existing permissions to match their most up-to-date descriptions.
* Added GrowthExperiments and GWToolset to the list, as they're present in some global groups
* Sorted A-Z
* Replace internal `{{int:...}}` keys in descriptions with their actual meanings and descriptions.

### CODE REVIEW

File: tool-labs/globalgroups/index.php [hunk 1]
```diff
@@ -212,12 +224,16 @@
     "right-translate-sandboxmanage" => "Manage sandboxed users",
 
     // WikimediaMessages extension: https://github.com/wikimedia/mediawiki-extensions-WikimediaMessages/blob/master/i18n/wikimedia/en.json
-    "right-editeditorprotected" => "Edit pages protected as \"{{int:protect-level-editeditorprotected}}\"",
-    "right-editextendedsemiprotected" => "Edit pages protected as \"{{int:protect-level-editextendedsemiprotected}}\"",
+    "right-banner-protect" => "Protect translatable messages handled by the Translate extension and included in CentralNotice banners",
+    "right-docseditor" => "Edit documentation"
```
Comment (Pathoschild) [inline tool-labs/globalgroups/index.php:228]: There's a missing comma at the end of this line:

Comment (MarcoAurelioWM) [reply tool-labs/globalgroups/index.php:228]: Oopsie. Fixing in a sec. Thanks for the review.

Comment (MarcoAurelioWM) [reply tool-labs/globalgroups/index.php]: Done.

**Review by Pathoschild — APPROVED** [review]

**Comment by Pathoschild** [comment]
Merged and deployed. Thanks!

**Pull request merged**